web-app/django/VIM/apps/instruments/views/instrument_list.py

```diff
@@ -1,10 +1,37 @@
 from django.db.models import Prefetch
-from django.db.models.query import QuerySet
 from django.views.generic import ListView
 from VIM.apps.instruments.models import Instrument, Language, InstrumentName
+import pysolr
 import requests
 
 
+# Helper classes to normalize Solr results
+class SolrInstrument:
+    def __init__(self, data, lang_code="en"):
+        self.pk = data.get("sid")
+        self.thumbnail = ThumbnailStub(data.get("thumbnail_url", None))
```

Comment:
Do you need a full class to store a simple string value? 

Why not: 

`self.thumbnail = data.get("thumbnail_url")`

(`None` is also the default if it doesn't exist, so `.get('foo', None)` and `.get('foo')` are equivalent.)